portfolio/src/main/webapp/script.js

```diff
@@ -73,17 +73,28 @@ function getComments() {
 }
 
 /*
+ * initializes comments on body load
+ * hides comments if user not logged in, otherwise 
  * extracts numComment from URL and then displays comment
  */
 function initComments() {
-  // extract numComment from URL 
-  var url = new URL(document.URL);
-  const numComments = url.searchParams.get('numComments') || 5;
+  fetch(`/loginStatus`).then(response => response.json()).then((msg) => {
+    if (msg.loginStatus == false) {
```

Comment:
nit: prefer to use === over == when comparing values. more info on why here: https://codeburst.io/javascript-double-equals-vs-triple-equals-61d4ce5a121a